homeassistant/components/denonavr/strings.json

```diff
@@ -0,0 +1,40 @@
+{
+  "config": {
+    "step": {
+      "user": {
+        "title": "Denon AVR Network Receivers",
+        "description": "Connect to your receiver, if the IP adress is not set, auto-discovery is used",
```

Comment:
Another `address` typo in the "description". Sorry, I haven't seen this before.

Reply:
fixed